src/Helper/OutputResultConverter.php

```diff
@@ -54,7 +54,11 @@ public static function convertOutputsToResult(array $outputs): JobResult
             }
             $uploadedArtifactsOutput = $output->getArtifactsUploaded();
             if ($uploadedArtifactsOutput) {
-                array_push($uploadedArtifacts, ...$uploadedArtifactsOutput);
+                array_push(
+                    $uploadedArtifacts,
+                    ...$uploadedArtifactsOutput['current'],
```

Comment:
nikde není řečený, že tam ten field je, to by musel bejt u getArtifactsUploaded arrayShape (což by teda mohl bejt)

Reply:
JJ, predelam to pak na objekt https://keboola.atlassian.net/browse/PS-3932